setup.py

```diff
@@ -85,7 +85,7 @@ def get_package_data(parent, roots):
 
 setup(
     name="pyfarm.agent",
-    version="0.8.3",
+    version="0.8.3.1",
```

Comment:
I did this version change because I was running git master on my agents, and I wanted to keep track of which agent already had this bugfix deployed. I can undo this agin if you object.